test/worker/sync/db-persist/mail-contributor.spec.js

```diff
@@ -21,7 +21,7 @@ chai.use(sinonChai);
 
 const MODULE_ID = path.relative(`${process.cwd()}/test`, module.id) || require(path.resolve('./package.json')).name;
 
-describe(MODULE_ID, function () {
+describe.only(MODULE_ID, function () {
```

Comment:
Left over `.only`

Reply:
Ugh, that's why I hate linter not working. Didn't even catch this on push.